src/components/RadioButton/__tests__/RadioButtonGroup.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import RadioButton from '../RadioButton';
+import RadioButtonGroup from '../RadioButtonGroup';
+
+describe('<RadioButtonGroup> that renders a group of radio buttons', () => {
+    let wrapper;
+    beforeEach(() => {
+        wrapper = mount(
+            <RadioButtonGroup name="group_name">
+                <RadioButton id="o1">Option 1</RadioButton>
+                <RadioButton id="o2">Option 2</RadioButton>
+            </RadioButtonGroup>
+        );
+    });
+
+    it('should render correctly', () => {
+        expect(toJson(wrapper)).toMatchSnapshot();
+    });
+    it('should pass name prop to children', () => {
+        wrapper
+            .find('RadioButton')
+            .forEach(option => expect(option.prop('name')).toEqual('group_name'));
+    });
+});
```

Comment:
Shall we add a test for checked item changes as well here?

Reply:
That's a native behaviour, not part of our code. So I think it is ok to leave it out. In Radio Button I added that test mostly to see that rest props are passed correctly.